[2018-06-04T13:13:16.450] [ERROR] Error - {"Error_Name":"Mongodb Connection Error","Error_Address":"Server.js - 31","Error_Message":{"name":"MongoNetworkError","message":"failed to connect to server [ds241699.mlab.com:41699] on first connect [MongoNetworkError: getaddrinfo ENOTFOUND ds241699.mlab.com ds241699.mlab.com:41699]"}}
[2018-06-04T13:13:16.466] [ERROR] Error - {"Error_Name":"","Error_Address":"","Error_Message":{"name":"MongoNetworkError","message":"failed to connect to server [ds241699.mlab.com:41699] on first connect [MongoNetworkError: getaddrinfo ENOTFOUND ds241699.mlab.com ds241699.mlab.com:41699]"}}
[2018-06-04T13:14:07.119] [ERROR] Error - {"Error_Name":"Mongodb Connection Error","Error_Address":"Server.js - 31","Error_Message":{"name":"MongoNetworkError","message":"failed to connect to server [ds241699.mlab.com:41699] on first connect [MongoNetworkError: getaddrinfo ENOTFOUND ds241699.mlab.com ds241699.mlab.com:41699]"}}
[2018-06-04T13:14:07.121] [ERROR] Error - {"Error_Name":"","Error_Address":"","Error_Message":{"name":"MongoNetworkError","message":"failed to connect to server [ds241699.mlab.com:41699] on first connect [MongoNetworkError: getaddrinfo ENOTFOUND ds241699.mlab.com ds241699.mlab.com:41699]"}}
[2018-06-04T17:07:29.751] [ERROR] Error - {"Error_Name":"","Error_Address":"","Error_Message":"SyntaxError: Unexpected token o in JSON at position 1"}
